packages/messages/src/use-cases/save-message.js

```diff
@@ -0,0 +1,32 @@
+// External dependencies
+const { Logger } = require('@booking-services/shared');
+
+const saveMessage = (MessageModel) => async ({
+  chatId,
+  text,
+}) => {
+  if (!MessageModel) {
+    Logger.error('[saveMessage]: Messages Model has not been injected');
+    throw new Error('Message model is needed');
+  }
+
+  if (typeof chatId !== 'string' || !chatId) {
+    Logger.error('[saveMessage]: Chat Id has not been provided');
+    throw new Error('Chat ID is needed');
+  }
+
+  if (typeof text !== 'string' || !text) {
+    Logger.error('[saveMessage] Text content has not been needed');
+    throw new Error('[saveMessage]: Text content is needed');
+  }
+
+  const message = new MessageModel({
+    chatId,
+    text,
+  });
+
+  return message.save()
```

Comment:
We are using async/await, so pls remove then/catch syntax and add this under a try/catch